ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Wallets.hs

```diff
@@ -190,6 +192,14 @@ deleteHdWallet pw rootId = do
             Keystore.delete (WalletIdHdSeq rootId) (pw ^. walletKeystore)
             return $ Right ()
 
+removeKeyFromKeyStore :: PassiveWallet
+                      -> [HD.HdRootId]
+                      -> IO ()
+removeKeyFromKeyStore pw rootIds =
```

Comment:
Why `key` (singular) if it removes a list of keys?
Also `s` is not capital in `Keystore`, so let's stick to this naming.